components/FormCreate.js

```diff
@@ -41,8 +54,8 @@ export default function FormCreate() {
       country: data.country,
       image: imageSearch.hits[increment].largeImageURL,
       description: data.description,
-      lng: data.lng,
-      lat: data.lat,
+      lng: lng,
```

Comment:
`lng: lng` when key and value has same name, then you can build your object like this                        
 const newActivity = {
      title: data.title,
      categoryFilter: categoryFilter,
      category: category,
      area: data.area,
      country: data.country,
      image: imageSearch.hits[increment].largeImageURL,
      description: data.description,
      lng,
      lat,
}
but no short hand notation for lng and lat